tests/rustdoc/intra-doc/link-to-current-crate.rs

```diff
@@ -0,0 +1,15 @@
+// Test to ensure that we can link to the current crate by using its name.
+
+#![deny(rustdoc::broken_intra_doc_links)]
+#![crate_name = "bar"]
+
+//@ has 'bar/index.html'
+//@ has - '//*[@class="docblock"]//a[@href="index.html"]' 'bar'
+//! [`bar`]
```

Comment:
Test case for `[crate@bar]`?

Reply:
There is no `crate` disambiguator. Can be added in a follow-up.